configure.ac

```
@@ -2462,6 +2464,15 @@ AS_IF([test x"$enable_frame_pointers" = "xyes"],
   [AC_MSG_NOTICE([not using frame pointers])
   frame_pointers=false])
 
+## arm64-specific options
+arm64_barrier_placement_default=FenceBeforeStore
+AC_ARG_ENABLE([alternate-arm64-code-generation],
+  [AS_HELP_STRING([--enable-alternate-arm64-code-generation],
```

Nit: Better to be explicit? 

```suggestion
  [AS_HELP_STRING([--enable-fence-before-store-arm64-codegen],
```

---

Changed to `--enable-branch-after-load-arm64-codegen` since `FenceBeforeStore` is the default.